daemon/cluster/noderunner.go

```diff
@@ -110,13 +115,21 @@ func (n *nodeRunner) start(conf nodeStartConfig) error {
 		joinAddr = conf.RemoteAddr
 	}
 
+	var defaultAddrPool []*net.IPNet
+	for _, address := range conf.DefaultAddressPool {
+		if _, b, err := net.ParseCIDR(address); err == nil {
```

Comment:
Could be done in a follow-up, but; I see other validation steps are done in `newNodeRunner()`; we should consider moving those all to a single place, not partly here, and partly there. We can also return parsing errors there.